deploy/kubernetes/snapshot-controller/rbac-snapshot-controller.yaml

```diff
@@ -9,13 +9,12 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: snapshot-controller
-  namespace: default # TODO: replace with the namespace you want for your controller, e.g. kube-system
+  namespace: kube-system
```

Comment:
This is used in CI so I'd like to ask Patrick to take a look.
@pohly Do you have any concerns for this namespace to be changed from "default" to "kube-system"?

Reply:
No objections.

Just beware that it might be necessary to change prow.sh so that it works with a rbac-snapshot-controller.yaml that has `namespace: default` (current stable release) and a newer rbac-snapshot-controller.yaml that has `namespace: kube-system` (canary) because the same prow.sh file is used with both, depending on the test job.